tests/spacetime/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+LINK_LIBRARIES(
+	spacetime-types
+	spacetime
+)
+
+ADD_CXXTEST(SpaceServerUTest)
```

Comment:
Because the SpaceServer code is put in the opencog/spacetime/, so I think I should put the test in the directory with the same name, e.g. tests/spacetime. But I think it's OK to put it in the tests/spatial.